test/wrapeth/provider_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule Wrapeth.ProviderTest do
+  use ExUnit.Case, async: true
+  import Mox
+
+  defmodule TestProvider do
+    use Wrapeth.Provider, otp_app: :wrapeth
+  end
+
+  # Make sure mocks are verified when the test exits
+  setup :verify_on_exit!
+
+  test "gets accounts" do
+    HttpMock
+    |> expect(:eth_accounts, fn _ -> {:ok, ["0x407d73d8a49eeb85d32cf465507dd71d507100c1"]} end)
+
+    assert TestProvider.eth_accounts() ==
+             ["0x407d73d8a49eeb85d32cf465507dd71d507100c1"]
```

Comment:
What about extracting repetitions to another variable? This way, you would only need to set it once.